util/headerreader/header_reader.go

```diff
@@ -313,7 +313,8 @@ func (s *HeaderReader) logIfHeaderIsOld() {
 	}
 	l1Timetamp := time.Unix(int64(storedHeader.Time), 0)
 	headerTime := time.Since(l1Timetamp)
-	if headerTime >= s.config().OldHeaderTimeout {
+	oldHeaderTimeout := time.Minute * 10
```

Comment:
Need to figure out how to override this in our test...

Reply:
Changing it here https://github.com/OffchainLabs/nitro/blob/master/util/headerreader/header_reader.go#L94 should do the trick i think.